types/office-js/index.d.ts

```diff
@@ -785,6 +791,8 @@ declare namespace Office {
         contentLanguage: string;
         /**
         * Gets information about the environment in which the add-in is running.
+        * 
+        * [Api set: ExcelApi 1.1, WordApi 1.1, PowerPointApi 1.1, Mailbox 1.5]
```

Comment:
Place Api set information in Remarks section and note alternative as Office.context.mailbox.diagnostics -- here and other instances

```suggestion
        * @remarks
        * [Api set: ExcelApi 1.1, WordApi 1.1, PowerPointApi 1.1, Mailbox 1.5]
        *
        * **Important**: For all Mailbox requirement sets, you can use the {@link https://docs.microsoft.com/javascript/api/outlook/office.mailbox?view=outlook-js-1.5&preserve-view-true#outlook-office-mailbox-diagnostics-member | Office.context.mailbox.diagnostics} property to get similar information.
```